### DISCUSSION TITLE
Open source the mobile client.

### DISCUSSION THREAD

**Discussion by ArjixWasTaken** [community/community#56359] [Mobile] [answered]
It would be awesome if the mobile app for GitHub was open source.

That way we could improve it wayyy more.

Is there a specific reason that it is not already open source?

**Comment by eliperkins** [comment, marked as answer]
Hey @ArjixWasTaken , I appreciate your feedback! Thanks for the idea, at this time we don't have plans to open source our mobile app. However, if you have ideas on how to improve it please continue to share that feedback in our Mobile category here on Discussions. ❤️

**Reply by ArjixWasTaken** [reply]
I'll reply with one improvement here:

Allow to make an issue with a template from the mobile app instead of opening the template in a browser.

And many more small stuff that can only be done from the website.
